railib/api.py

```diff
@@ -34,6 +35,9 @@
 PATH_USER = "/users"
 PATH_OAUTH_CLIENT = "/oauth-clients"
 
+# logger
+logger = logging.getLogger("rai")
```

Comment:
Everything looks good except the way we  compute the logger name, we should compute it from __name__ of the top level module (railib in this case)

Reply:
might be helpful: https://stackoverflow.com/questions/8706309/how-to-reference-to-the-top-level-module-in-python-inside-a-package

Reply:
using `__package__` global variable to solve the requirement